execution/gethexec/sequencer.go

```diff
@@ -1119,11 +1234,43 @@ func (s *Sequencer) Start(ctxIn context.Context) error {
 		return 0
 	})
 
+	if config.Timeboost.Enable {
+		s.StartExpressLane(
+			ctxIn,
+			common.HexToAddress(config.Timeboost.AuctionContractAddress),
+			common.HexToAddress(config.Timeboost.AuctioneerAddress))
+	}
+
 	return nil
 }
 
+func (s *Sequencer) StartExpressLane(ctx context.Context, auctionContractAddr common.Address, auctioneerAddr common.Address) {
+	if !s.config().Timeboost.Enable {
+		log.Crit("Timeboost is not enabled, but StartExpressLane was called")
+	}
+	rpcClient, err := rpc.DialContext(ctx, s.config().Timeboost.SequencerHTTPEndpoint)
```

Comment:
I really don't like this.
expresslaneservice is inside execution and has direct access to the blockchain, no reason to go through network.

Reply:
Addressed in https://github.com/OffchainLabs/nitro/pull/2762

Reply:
Merged into this branch.